pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1301,14 +1301,21 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 		return
 	}
 
+	// Use StorageClass.ReclaimPolicy or fall back to Delete if it's somehow blank
+	reclaimPolicy := storageClass.ReclaimPolicy
+	if reclaimPolicy == "" {
```

Comment:
None really, just paranoia (see https://github.com/kubernetes/kubernetes/pull/47987#discussion_r123922889) . But now that it is a ptr I think we should check anyway before derefing.